Mathlib/NumberTheory/ArithmeticFunction.lean

```diff
@@ -406,7 +402,6 @@ theorem zeta_apply {x : ℕ} : ζ x = if x = 0 then 0 else 1 :=
 theorem zeta_apply_ne {x : ℕ} (h : x ≠ 0) : ζ x = 1 :=
   if_neg h
 
--- Porting note: removed `@[simp]`, LHS not in normal form
 theorem coe_zeta_smul_apply {M} [Semiring R] [AddCommMonoid M] [Module R M]
```

Comment:
Do we know what the normal form is here? It would be a real shame if simp couldn't simplify the left here, I'd certainly hope it can!

Reply:
Restored.